beacon-chain/rpc/service.go

```diff
@@ -293,6 +293,9 @@ func (s *Service) Start() {
 		HeadFetcher:               s.cfg.HeadFetcher,
 		ExecutionChainInfoFetcher: s.cfg.ExecutionChainInfoFetcher,
 	}
+	s.cfg.Router.HandleFunc("/eth/v1/node/trusted_peers", nodeServerV1.ListTrustedPeer).Methods("GET")
```

Comment:
I don't think using the same namespace as the official beacon-apis is a good idea, we ideally want to differentiate official api endpoints vs internal ones. cc @rkapka

Reply:
Agreed, we should change this to `/prysm/node/trusted_peers`. No need for the `v1` part because it's a versioning scheme adapted by the official API spec, which we do not necessarily want to follow.